src/lib/pages/module-details/components/tables/history/PublishedEventsTableRow.tsx

```diff
@@ -39,7 +39,7 @@ export const PolicyChanges = ({ history }: { history: ModuleHistory }) => {
           <Text variant="body2" color="text.dark">
             Changed from{" "}
             <Text as="span" fontWeight={700}>
-              {capitalize(history.previousPolicy)}
+              {capitalize(history.previousPolicy ?? undefined)}
```

Comment:
remove `?? undefined`

Reply:
I also made a change from switch case to if/else. Please re-review. Thank!